qutip/solve/pdpsolve.py

```diff
@@ -200,6 +197,9 @@ def __init__(self, H=None, state0=None, times=None, c_ops=[], sc_ops=[],
         else:
             self.map_func = serial_map
 
+        if progress_bar is None:
+            self.progress_bar = TextProgressBar(self.ntraj)
```

Comment:
Now that the progress bar constructor starts the progress bar, we should construct the progress bar where it is used.

Reply:
This file is removed in #2131.
As well as `solve/stochastic.py`.